src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/Organisations/OdsOrganisationViewModel.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace NHSD.BuyingCatalogue.Organisations.Api.ViewModels.Organisations
 {
-    public sealed class OdsViewModel
+    public sealed class OdsOrganisationViewModel
```

Comment:
I think we agreed we weren't going to call these `ViewModel` where they aren't used in the context of a view. However, I don't think we agreed what we would call them instead! As I suspect we'll have a tech debt task to rename these I guess this can be left as is for now.

Reply:
😄 I'm glad you've mentioned this. I'm aware of our meeting and what we agreed on, however when it came to naming it, I didn't want to pave the way forward without consulting with the team! Hence why I left it with the wrong naming scheme. I was counting on the TD task to rename all of them in one fell swoop, as we probably don't want to have half of 'ViewModels' named one way and the rest a different way.

I'll be leaving it as is for now until we rename them 👍